src/main/kotlin/com/mparticle/kits/GoogleAnalyticsFirebaseGA4Kit.kt

```diff
@@ -464,16 +477,39 @@ class GoogleAnalyticsFirebaseGA4Kit : KitIntegration(), KitIntegration.EventList
         }
 
     }
+    private fun hashMapToBundle(hashMap: HashMap<String, Any>): PickyBundle {
```

Comment:
```suggestion

    private fun hashMapToBundle(hashMap: HashMap<String, Any>): PickyBundle {
```
Missing new line before function